Automattic/wp-calypso
# Gutenboarding: Remove page-templates.json

#### Changes proposed in this Pull Request

This file seems unused.

#### Testing instructions

Make sure this file isn't used anywhere in Calypso, and that Gutenboarding runs properly.

Note that this file _was_ used for a while outside of Calypso, in a REST API endpoint that we use for Gutenboarding. However, I believe that I removed the only part of code where it was evaluated in D41177-code, and @sirreal removed the LOCs that loaded it in D41281-code (Ctr+F `page-templates` in both diffs).

This means we'll need to exercise some extra caution when landing this (test it properly while it's on staging etc).
